core/templates/pages/exploration-player-page/services/hint-and-solution-modal.service.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Oppia Authors. All Rights Reserved.
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
We usually only update this in new files.... You can probably remove this change

Reply:
done!